### DISCUSSION TITLE
Launching the Machine Learning Category 🧠!!

### DISCUSSION THREAD

**Discussion by sethjuarez** [community/community#51614] [Programming Help]
Machine Learning projects and models are pretty awesome. Turns out that a lot of folks happen to put their ML code on GitHub and even use some of the amazing GitHub features to manage and power their projects. Most of the questions related to this amazing area tended to fall in a sprinkling of different categories so we thought we would try to consolidate!

Before you post, please review our [Community Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) to make sure we keep this a friendly and vibrant place. 

This category will be a trial from April 1st to September 30th, 2023 and we may be checking in with those in the community to see how they’re feeling about it, but your feedback is welcome at any time 👂! 

Happy posting 💽!

**Comment by aws11142** [comment]
Logan and POST the UOTH answer WWW.GIWA.COM

**Reply by aws11142** [reply]
awesome. Turns out that a lot of folks happen to put their ML code on GitHub and even use some of the amazing GitHub features to manage and power their projects. Most of the questions related to this amazing area tended to fall in a sprinkling of different categories so we thought we would try to consolidate!

Before you post, please review our [Community Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct)

**Reply by ErickWDaniel** [reply]
Thats so true

**Comment by aws11142** [comment]
> Logan and POST the UOTH answer WWW.GIWA.COM

awesome. Turns out that a lot of folks happen to put their ML code on GitHub and even use some of the amazing GitHub features to manage and power their projects. Most of the questions related to this amazing area tended to fall in a sprinkling of different categories so we thought we would try to consolidate!

Before you post, please review our [Community Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct)

**Comment by Dave4334** [comment]
Not sure yet

**Comment by aws11142** [comment]
<https://fedoraproject.org/wiki/Licensing:MIT#Modern_Style_with_sublicense>.

"Submit" means to use some mode of digital communication (for example,

without limitation, mailing lists, bug tracking systems, and source

code version control systems administered by Fedora) to voluntarily

provide a Contribution to Fedora.

"Unlicensed" means not Licensed.

"Work" means a copyrightable work of authorship. A Work may be a

portion of a larger Work, and a Work may be a modification of or

addition to another Work.

"You" means the individual accepting this instance of the FPCA.

1. Copyright Permission Required for All Contributions.

If You are not the copyright holder of a given Contribution that You

wish to Submit to Fedora (for example, if Your employer or university

holds copyright in it), it is Your responsibility to first obtain

authorization from the copyright holder to Submit the Contribution

**Comment by aws11142** [comment]
> https://fedoraproject.org/wiki/Licensing:MIT#Modern_Style_with_sublicense.
> 
> "Submit" means to use some mode of digital communication (for example,
> 
> without limitation, mailing lists, bug tracking systems, and source
> 
> code version control systems administered by Fedora) to voluntarily
> 
> provide a Contribution to Fedora.
> 
> "Unlicensed" means not Licensed.
> 
> "Work" means a copyrightable work of authorship. A Work may be a
> 
> portion of a larger Work, and a Work may be a modification of or
> 
> addition to another Work.
> 
> "You" means the individual accepting this instance of the FPCA.
> 
> 1. Copyright Permission Required for All Contributions.
> 
> If You are not the copyright holder of a given Contribution that You
> 
> wish to Submit to Fedora (for example, if Your employer or university
> 
> holds copyright in it), it is Your responsibility to first obtain
> 
> authorization from the copyright holder to Submit the Contribution

**Comment by AaronCWacker** [comment]
Thanks for adding this.  I found this helpful and ask to continue after trial.  BTW - enjoying your Youtube videos about PromptFlow and have been enjoying it with the source and examples.  My playlist which includes it and other tutorials is here:  https://www.youtube.com/playlist?list=PLHgX2IExbFot1nzSmFSRHAsKJNRzZUKDh  Azure AI Studio and Promptflow have been my most useful tools this month.  I was interested if you might do a blog next on youtube about AutoGen which is my next favorite MS git repo for AI and ML!  Have a great day and thanks for all the fish.

**Comment by khaledbkheet** [comment]
> Machine Learning projects and models are pretty awesome. Turns out that a lot of folks happen to put their ML code on GitHub and even use some of the amazing GitHub features to manage and power their projects. Most of the questions related to this amazing area tended to fall in a sprinkling of different categories so we thought we would try to consolidate!
> 
> Before you post, please review our [Community Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) to make sure we keep this a friendly and vibrant place.
> 
> This category will be a trial from April 1st to September 30th, 2023 and we may be checking in with those in the community to see how they’re feeling about it, but your feedback is welcome at any time 👂!
> 
> Happy posting 💽!

**Comment by queenofcorgis** [comment]
Please refer to this announcement for the [Machine Learning and ML Ops category 🧠 ](https://github.com/orgs/community/discussions/69839)
